README.adoc

```diff
@@ -145,306 +136,184 @@ The [hotspot=from file=0]`FROM` instruction initializes a new build stage and in
-docker build -t ol-runtime --no-cache=true .
-```
 
-Learn more about the image build process on the https://docs.docker.com/engine/reference/commandline/image_build[Docker documentation^].
+To view a full list of all available containers, you can run the `docker ps -a` command.
 
+If your container is running without problems, point your browser to http://localhost:9080/system/properties[http://localhost:9080/system/properties^] 
+where you can see a JSON file that contains the system properties of the JVM in your container.
 
 // =================================================================================================
-// Running your application in Docker container
+// Updating the application while the container is running 
 // =================================================================================================
 
-== Running your application in Docker container
-
-Now that your image is built, execute the Docker `run` command with the absolute path to this guide:
-
-include::{common-includes}/os-tabs.adoc[]
-
-[.tab_content.windows_section]
---
-Make sure you substitute the absolute path to this guide.
-
-[role='command']
-```
-docker run -d --name rest-app -p 9080:9080 -p 9443:9443 -v <absolute path to guide>/start/target/liberty/wlp/usr/servers:/servers ol-runtime
-```
---
-
+== Updating the application while the container is running
 
-[.tab_content.mac_section]
---
-[role='command']
-```
-docker run -d --name rest-app -p 9080:9080 -p 9443:9443 -v $(pwd)/target/liberty/wlp/usr/servers:/servers ol-runtime
-```
---
+// File 0 
+PropertiesResource.java
+[source, java, linenums, role='code_column hide_tags=comment']
+----
+include::finish/src/main/java/io/openliberty/guides/rest/PropertiesResource.java[]
+----
 
+With your container running, make the following update to the source code:
 
-[.tab_content.linux_section]
---
-Use the `-u` flag to override the default user within the container to match the user 
-id on your host machine. This is to ensure the default user has correct permissions to the 
-server directory. 
+[role="code_command hotspot hotspot file=0", subs="quotes"]
+----
+#Update the `PropertiesResource` class.#
+`src/main/java/io/openliberty/guides/rest/PropertiesResource.java`
+----
 
-[role='command']
-```
-docker run -d --name rest-app -p 9080:9080 -p 9443:9443 -v $(pwd)/target/liberty/wlp/usr/servers:/servers -u `id -u` ol-runtime
-```
---
+[role="edit_command_text"]
+Change the endpoint of your application from `properties` to `properties-new` by changing the [hotspot=Path]`@Path`
+annotation to `"properties-new"`.
 
-Alternatively, you can also execute the `run` command to mount a single server instead of the whole `servers` directory:
-[role='command']
+After you make the file changes, Open Liberty automatically updates the application. To see these changes reflected in the application, 
+point your browser to http://localhost:9080/system/properties-new[http://localhost:9080/system/properties-new^].
```

Comment:
See previous comment about `404` error and app running on port 9081

Reply:
Resolved, had an application on my machine running on port 9080 for some reason. Terminated this application and restarted guide app. Everything worked as expected.